### DISCUSSION TITLE
Import Popper

### DISCUSSION THREAD

**Discussion by kozmik-moore** [helloflask/bootstrap-flask#273] [Q&A]
I see that there are references to Popper in the Bootstrap5 class but I do not understand how to import it.

Do I use ```bootstrap.js()``` to import it?

My version of Popper is installed locally using npm.

Thanks.

**Comment by PanderMusubi** [comment]
Popper is shipped with Bootstrap-Flask, see https://github.com/helloflask/bootstrap-flask/tree/master/flask_bootstrap/static/bootstrap5/umd
